include/text.php

```diff
@@ -652,7 +647,7 @@ function logger($msg, $level = 0) {
 
 	$logline = sprintf("%s@%s\t[%s]:%s:%s:%s\t%s\n",
 			DateTimeFormat::utcNow(DateTimeFormat::ATOM),
-			$process_id,
+			Process::getCode(),
```

Comment:
Maybe rename `Process:getCode()` to `Process:getProcessId()`?

Reply:
I agree it's confusing as well, but the `code` is supposed to be an identifier string that would be added to logs. It uses the process id but not only. I don't have a better name for it though, feel free to suggest another, but `getProcessId` is inaccurate.

Reply:
What about Process::toString() ?

Reply:
That's what I initially suggested, but I'm not sure how it would look. Maybe
```php
$processCode = (string) Process::getInstance();
```

Reply:
I think this doesn't look very intuitive :-( .. Why not:
```php
$processCode = Process::toString();
```
It's shorter, explicit and more intuitive

Reply:
Fair enough, I'm all for it.